bin/capsule-certs-generate

```diff
@@ -31,6 +31,7 @@ CONFIG_CONTENT = <<EOF
 :answer_file: #{answers_file.path}
 :installer_dir: #{INSTALLER_DIR}
 :installer_name: 'foreman-installer'
+:product_name: 'Katello'
```

Comment:
`product_name` seems a bit on the nose, since we are dealing with scenarios -- why not use the name defined in the config file for the scenario?